app/src/main/java/com/nervousfish/nervousfish/modules/cryptography/KeyGeneratorAdapter.java

```diff
@@ -28,4 +37,25 @@ private KeyGeneratorAdapter(final IServiceLocatorCreator serviceLocatorCreator)
     public static ModuleWrapper<KeyGeneratorAdapter> newInstance(final IServiceLocatorCreator serviceLocatorCreator) {
         return new ModuleWrapper<>(new KeyGeneratorAdapter(serviceLocatorCreator));
     }
+
+    /**
+     * Generates a random KeyPair with the RSA algorithm.
+     * @return a randomly generated KeyPair
+     */
+    public static KeyPair generateRSAKeyPair() throws NoSuchAlgorithmException, InvalidKeySpecException {
+        final KeyPairGenerator k = KeyPairGenerator.getInstance("RSA");
+        k.initialize(2048);
+
+        final java.security.KeyPair keyPair = k.generateKeyPair();
+        final KeyFactory fact = KeyFactory.getInstance("RSA");
```

Comment:
Could you use the variable names `key` instead of `k` and `keyFactory` instead of `fact`